ts/screens/profile/DeveloperModeSection.tsx

```diff
@@ -340,6 +360,18 @@ const DesignSystemSection = () => {
         value={isNewScanSectionLocallyEnabled}
         onSwitchValueChange={onNewScanSectionToggle}
       />
+      {/* this control isFastLoginSessionRefreshRemoteFFActive is a
+      workaround to hide this toogle before this task
+      (https://pagopa.atlassian.net/browse/IOPID-2051)
+      is completed because otherwise nothing would be activated using this toogle
+       */}
+      {isFastLoginSessionRefreshRemoteFFActive && (
```

Comment:
You did it right adding the toggle value in `OR` here:
`isFastLoginEnabled && (sessionRefresh || isSessionRefreshToggleActive)`
so what the point to limit it with the remote FF? The toggle'd always be visible, am I right? I mean, even if the remote FF is not enabled, we want to have the possibility to force the new refresh handling feature under the dev mode. What do you think?

Reply:
Of course, the toggle is always visible, **but** only in dev mode and **not** everyone uses dev mode. The remote FF will then be used to activate the functionality on **all** users' devices

Reply:
Okay, got it.